xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/notification/email/macros.vm

```diff
@@ -95,7 +95,7 @@
 
 #macro(displayNotificationPage $event)
   <div>
-    <div style="background-color: #f5f5f5; color: #777777; padding: 4px 8px; border-radius: 7px; font-size: 8px;">
+    <div style="background-color: #f5f5f5; color: #707070; padding: 4px 8px; border-radius: 7px; font-size: 8px;">
```

Comment:
Maybe we should use instead velocity variables defined in this file: so that in the future if we want to change those colors we only need to change the variables. Advantage is also that it would allow to reuse those colors in custom templates.

Reply:
Addressed in the latest commits :+1: